### DISCUSSION TITLE
News Cache [2024-01-15]

### DISCUSSION THREAD

**Discussion by amyegan** [vercel/community#5483] [Announcements]
### 🚢  Ships

- [Vercel Firewall proactively protects against vulnerability in the Clerk SDK](https://vercel.com/changelog/vercel-firewall-proactively-protects-against-vulnerability-in-the-clerk-sdk)
- [Easier upgrades from free to paid teams](https://vercel.com/changelog/2024-01-account-changes)
- [Login with passkey is now supported](https://vercel.com/changelog/login-with-passkey-is-now-supported)
- [HTTPS DNS records are now supported in Vercel DNS](https://vercel.com/changelog/https-dns-records-are-now-supported-in-vercel-dns)

### 📚 Education

- [Common mistakes with the Next.js App Router and how to fix them](https://vercel.com/blog/common-mistakes-with-the-next-js-app-router-and-how-to-fix-them)
- [Learn how to fetch, cache, and revalidate data using the Next.js App Router](https://x.com/nextjs/status/1746913546003243211)
- [10 common mistakes with the Next.js App Router](https://youtu.be/RBM03RihZVs)

### ℹ️ Reminders

- Accounts are being migrated to Vercel’s new billing system. When your account is moved, you will receive [two invoices associated with two charges in the same subscription period](https://vercel.com/docs/accounts/plans/pro/billing#why-did-i-get-two-invoices-in-the-same-subscription-period). These are not duplicate and you will not have to pay anything extra. If you have any concerns with your invoice, you can always [reach out to the Customer Success Team](https://vercel.com/help) for help.
- [Node.js 16 deprecation](https://vercel.com/changelog/node-js-16-deprecation) and [Node.js v20 LTS in beta](https://vercel.com/changelog/nodejs-20)

**Comment by bdxoul** [comment]
Hello there, 

My domain provider doesn't have the option to use txt records, I haven't used this account on any vercel account please help , 
![image](https://github.com/vercel/community/assets/12608621/58d3f438-0873-4347-918c-61d136892fac)

My other Domains got instant accepted by Vercel. It's been 2 days . i have been trying , I use txt records using cloudlfare but nothing happens
